### PR TITLE
DevEx: Validate businessType, Files must be Files

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/entities/cases/CaseExternal.js [hunk 1]
```diff
@@ -96,7 +101,7 @@ CaseExternal.commonRequirements = {
     .string()
     .valid(...Object.values(PARTY_TYPES))
     .required(),
-  petitionFile: joi.object().required(), // TODO: object definition
+  petitionFile: joi.object().instance(File).required(),
```
Comment (rachelschneiderman) [inline shared/src/business/entities/cases/CaseExternal.js:108]: 🎉

File: shared/src/business/entities/cases/CaseExternal.js [hunk 2]
```diff
@@ -118,7 +123,7 @@ CaseExternal.commonRequirements = {
     .string()
     .valid(...PROCEDURE_TYPES)
     .required(),
-  stinFile: joi.object().required(), // TODO: object definition
+  stinFile: joi.object().instance(File).required(),
```
Comment (rachelschneiderman) [inline shared/src/business/entities/cases/CaseExternal.js:130]: 👏

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/entities/cases/CaseExternal.js (hunk 1 above)
Comment (kkoskelin) [reply shared/src/business/entities/cases/CaseExternal.js:108]: I'm wondering whether `File` is available in the node environment and will this work as we want it to?  Or should this be expanded to instead be `Blob`?

File: web-client/src/utilities/queryStringDecoder.test.js [hunk 3]
```diff
@@ -1,6 +1,18 @@
 import { queryStringDecoder } from './queryStringDecoder';
 
 describe('queryStringDecoder', () => {
+  beforeAll(() => {
+    global.window = Object.create(window);
+    const url =
+      'http://dummy.com/?code=CODE&token=TOKEN&path=BoulevardOfBrokenMemes';
```
Comment (kkoskelin) [inline web-client/src/utilities/queryStringDecoder.test.js]: example.com please

**Review by kkoskelin — APPROVED** [review]
Fix the test url and g2g.

**Pull request closed without merging**